src/background.ts

```diff
@@ -76,8 +106,9 @@
 ): Promise<T> => {
   try {
     await browser.storage.session.set({ [host]: cache });
-  } catch {
+  } catch (e) {
     // Quota exceeded
+    console.info(e);
```

Comment:
Errors generated by `browser.storage.session.set()` are not defined in individual classes, so they cannot be checked using `instanceof`. Since there seems to be no errors other than exceeding the quota if the arguments and manifest settings are not mistaken, this is the current implementation.